image-search/ViewControllers/ViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ViewController.swift
+//  image-search
+//
+//  Created by Renato Vieira on 9/25/19.
+//  Copyright © 2019 Stant. All rights reserved.
+//
+import Foundation
+import UIKit
+
+class ViewController: UIViewController {
+    
+    let loading = ActivytyIndicator()
+    
+    @IBOutlet weak var imageBlur       : UIImageView!
+    @IBOutlet weak var image           : UIImageView!
+    @IBOutlet weak var labelNameAlbum  : UILabel!
+    @IBOutlet weak var labelAlbumCover : UILabel!
+    @IBOutlet weak var labelNameArtist : UILabel!
+    @IBOutlet weak var button          : UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        button.layer.cornerRadius = 27
+        self.getAlbum()
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        loading.view.frame = self.view.safeAreaLayoutGuide.layoutFrame
+    }
+    
+    @IBAction func alterText(_ sender: Any) {
+        self.getAlbum()
+    }
+    
+    func getAlbum() {
+        requestAlbum (url: "", view: self)
```

Comment:
If you are always passing an empty string (I checked) and your URL is setted on requestAlbum function, you can remove this parameter of "url" and only pass the view.

Reply:
Done